tools/topology/development/sof-tgl-nocodec-ci.m4

```diff
@@ -156,6 +135,21 @@ PIPELINE_PCM_ADD(sof/pipe-volume-capture.m4,
 	1000, 0, 0,
 	48000, 48000, 48000)
 
+# Capture pipeline 7 from demux on PCM 6 using max 2 channels of s32le.
+PIPELINE_PCM_ADD(sof/pipe-passthrough-capture-sched.m4,
```

Comment:
This looks like we are also adding new pipelines ? Separate commit.

Reply:
> This looks like we are also adding new pipelines ? Separate commit.

This is just a movement of the existed one to make sure the PIPELINE_PCM_ADD()s are added in pipeline ID ascending sequence.